src/v2/ion/i18nTransformer.js

```diff
@@ -137,6 +137,10 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   // Remap authn API errors to OIE
   'api.authn.poll.error.push_rejected': 'oktaverify.rejected',
+
+  // Remap duo API errors to OIE
+  'oie.authenticator.duo.method.duo.verification_timeout' : 'oie.authenticator.duo.error',
+  'oie.authenticator.duo.method.duo.verification_failed' : 'oie.authenticator.duo.error'
```

Comment:
👏🏼 This should be the pattern we try to abide to moving forward. Not only do we have a nice `i18n` key to reference, but customizations can be applied to only a single key!